lib/repl.js

```diff
@@ -1126,7 +1126,7 @@ function isRecoverableError(e, self) {
       self._inTemplateLiteral = true;
       return true;
     }
-    return /^(Unexpected end of input|Unexpected token)/.test(message);
+    return /^(Unexpected end of input|Unexpected token|missing \) after argument list)/.test(message);
```

Comment:
@Zirak mmm, then lets do it as multiple regexps. I am actually eager to see that solution, as I didn't understand it :D